### PR TITLE
Improve README.md installation, other formatting

### PR DESCRIPTION
**Pull request by jtmoon79**
Improve README.md installation instructions. Fix ordering, adding section for Visual Studio individual components.

Fix other miscellaneous formatting for file references.

### CODE REVIEW

**Comment by WinProtoTest** [comment]
Thanks for the pull request. We'll review it as soon as possible.

File: README.md [hunk 1]
```diff
@@ -44,30 +44,30 @@ You should install the software listed below based on your testing purpose, incl
...
+   1. _Turn Windows features on or off_
+   1. ☑ _.NET Framework 3.5 (includes .NET 2.0 and 3.0)
    This is necessary for WiX Toolset.
-1. [WiX Toolset v3.11](https://github.com/wixtoolset/wix3/releases/tag/wix3111rtm)
-1. [WiX Toolset Visual Studio 2017 Extension](https://marketplace.visualstudio.com/items?itemName=RobMensching.WixToolsetVisualStudio2017Extension)
-   It is required if your want to build test suites or Protocol Test Manager from source code.
-1. [Microsoft Message Analyzer](https://www.microsoft.com/en-us/download/details.aspx?id=44226)
+1. [WiX Toolset v3.11](https://github.com/wixtoolset/wix3/releases/tag/wix3111rtm)<br />
+1. [WiX Toolset Visual Studio 2017 Extension](https://marketplace.visualstudio.com/items?itemName=RobMensching.WixToolsetVisualStudio2017Extension)<br />
+   WiX Toolset components re required if your want to build test suites or Protocol Test Manager from source code.
```
Comment (chenlu0616) [inline README.md]: Redundant "re" here

File: README.md [hunk 2]
```diff
@@ -44,30 +44,30 @@ You should install the software listed below based on your testing purpose, incl
...
-1. [Spec Explorer 2010 v3.5.3146.0](https://visualstudiogallery.msdn.microsoft.com/271d0904-f178-4ce9-956b-d9bfa4902745/)
+1. [Spec Explorer 2010 v3.5.3146.0](https://visualstudiogallery.msdn.microsoft.com/271d0904-f178-4ce9-956b-d9bfa4902745/)<br />
    It is only required if you want to build or run the test suites that contain Model-Based Test cases. If you want to regenerate Model-Based Test cases, you must install Visual Studio 2012.
-1. [Protocol Test Framework build 1.0 (build 1.0.6500.0)](https://github.com/Microsoft/ProtocolTestFramework/releases/tag/1.0.6500.0)
+1. [Protocol Test Framework build 1.0 (build 1.0.6500.0)](https://github.com/Microsoft/ProtocolTestFramework/releases/tag/1.0.6500.0)<br />
    You can use a released MSI file or build it from source code.
-1. Enable .NET Framework 3.5.1
-   _Turn Windows features on or off_
-   ☑ _.NET Framework 3.5 (includes .NET 2.0 and 3.0)
+1. Enable .NET Framework 3.5.1<br/>
```
Comment (sweeetday) [inline README.md]: [Enable .NET Framework 3.5.1]  instead

Comment (jtmoon79) [reply README.md]: Do you mean:

    1. Enable .NET Framework 3.5.1<br/>
       1. _Turn Windows features on or off_
       1. check _.NET Framework 3.5 (includes .NET 2.0 and 3.0)_
       This is necessary for WiX Toolset.

?

Comment (jtmoon79) [reply README.md]: ping @sweeetday

Comment (sweeetday) [reply README.md]: Hi, Jtmoon79
Yes, the short set is preferred.
Thanks.

**Comment by jtmoon79** [comment]
Ping @sweeetday

**Pull request merged**